tests/test_cases/test_force_release/test_force_release.py

```diff
@@ -5,7 +5,8 @@
 from cocotb.triggers import Timer
 
 
-@cocotb.test(expect_fail=cocotb.SIM_NAME.lower().startswith("ghdl"))
+# Verilator does not observe the vpi_put_value() flags field
+@cocotb.test(expect_fail=cocotb.SIM_NAME.lower().startswith(("ghdl", "verilator")))
```

Comment:
I wish we also knew why GHDL doesn't work so we could add a github issue label now before we forget, but oh well.